exercises/flatten-array/makefile

```diff
@@ -0,0 +1,28 @@
+CFLAGS  = -std=c99
+CFLAGS += -g
+CFLAGS += -Wall
+CFLAGS += -Wextra
+CFLAGS += -pedantic
+CFLAGS += -Werror
+
+# https://stackoverflow.com/questions/11551986/gcc-missing-braces-around-initializer
```

Comment:
Date this and note the current SO license, if you could. Might also be worth including the bug report link  to cover the case that (gasp!) SO is gone/down at some point in the future.

Reply:
I did not forgot your suggestion from the 'bracket-push' commit, but in this case I did not copy anything from stackoverflow.

Will change the comment in the makefile to point directly to bugzilla.